js/modules/k6/http/http.go

```diff
@@ -127,3 +178,11 @@ func (*HTTP) URL(parts []string, pieces ...string) (httpext.URL, error) {
 	}
 	return httpext.NewURL(urlstr, name)
 }
+
+// Client represents a stand-alone HTTP client.
+//
+// TODO: move to its own file
+type Client struct {
+	moduleInstance   *ModuleInstance
```

Comment:
This is a bit long maybe just `mi`

Reply:
Considering @na--'s point, I also prefer `moduleInstance`, and would avoid abbreviating for clarity's sake.

Reply:
Could be just `module` be an option? Like `c.module.vu.State()`

Reply:
Hmm :thinking: It would be a bit misleading, since we technically have the `Root` module that makes module instances per VU, but with the new architecture that shouldn't really be embedded in anything besides the module instances themselves, so it would probably be fine :man_shrugging:

Reply:
I still kind of feel that `mi` will likely be pretty common occurrence as a name across internal modules that it will not be ... too hard to know what it is, you already use it , just not for the field ;) 

But we can always change it later